<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>631</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 25pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;
    border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Downloads/corporate_Music-Flex-Icons-03.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>321</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 100%;
    height: 100%;
    background-color: rgba(0, 0, 0, 0.5); /* Nền đen với độ trong suốt 50% */
    display: flex;
    justify-content: center;
    align-items: center;
border-radius: 20px</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;
    border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../img/corporate_Music-Flex-Icons-03.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_eye_cp">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>370</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../img/eye-slash-solid.svg</normaloff>../img/eye-slash-solid.svg</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_eye_p">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../img/eye-slash-solid.svg</normaloff>../img/eye-slash-solid.svg</iconset>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>281</width>
        <height>471</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Segoe Print&quot;;</string>
         </property>
         <property name="text">
          <string>Email:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_email">
         <property name="styleSheet">
          <string notr="true">width: 100%;
  padding: 10px;
  font-size: 14px;
  border: 1px solid #B3B3B3; 
  border-radius: 20px; 
  background-color: transparent;
  color: rgb(0, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Segoe Print&quot;;</string>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_name">
         <property name="styleSheet">
          <string notr="true">width: 100%;
  padding: 10px;
  font-size: 14px;
  border: 1px solid #B3B3B3; 
  border-radius: 20px; 
  background-color: transparent;
  color: rgb(0, 0, 0) </string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">color:rgb(255, 255, 255);
font: 15pt &quot;Segoe Print&quot;;</string>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_password">
         <property name="styleSheet">
          <string notr="true">width: 100%;
  padding: 10px;
  font-size: 14px;
  border: 1px solid #B3B3B3; 
  border-radius: 20px; 
  background-color: transparent;
  color: rgb(0, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 15pt &quot;Segoe Print&quot;;</string>
         </property>
         <property name="text">
          <string>Confirm password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_conf_pwd">
         <property name="styleSheet">
          <string notr="true">width: 100%;
  padding: 10px;
  font-size: 14px;
  border: 1px solid #B3B3B3; 
  border-radius: 20px; 
  background-color: transparent;
  color: rgb(0, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::EchoMode::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Confirm Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_register">
         <property name="styleSheet">
          <string notr="true">background: qlineargradient( spread: pad, x1: 0, y1: 0, x2: 1, y2: 0, stop: 0 #fff9b0, stop: 1 #041f60 );
font: 9pt &quot;Segoe Print&quot;;
border: 2px solid #8A2BE2; /* Viền màu tím nhạt hơn */
 color: white; /* Màu chữ trắng */
 padding: 10px 20px; 
text-align: center; 
text-decoration: none; 
display: inline-block;
 font-size: 16px; 
margin: 4px 2px; 
cursor: pointer;
border-radius: 20px; /* Cạnh bo tròn */ 
box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19); /* Đổ bóng */
transition: all 0.3s ease; /* Hiệu ứng chuyển đổi */</string>
         </property>
         <property name="text">
          <string>Sign Up</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="styleSheet">
            <string notr="true">padding: 10px;
font-size: 14px;
border-radius: 20px; 
background-color: transparent;
color:rgb(0, 0, 0);
font: 9pt &quot;Segoe Print&quot;;</string>
           </property>
           <property name="text">
            <string>Have an account already? </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_login">
           <property name="styleSheet">
            <string notr="true">border-radius: 15px;
font: 9pt &quot;Segoe Print&quot;;
color: rgb(255, 0, 0)</string>
           </property>
           <property name="text">
            <string>Sign In</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>label_14</zorder>
     <zorder>label_7</zorder>
     <zorder>label_5</zorder>
     <zorder>label_3</zorder>
     <zorder>verticalLayoutWidget</zorder>
     <zorder>btn_eye_cp</zorder>
     <zorder>btn_eye_p</zorder>
    </widget>
    <widget class="QWidget" name="page_3"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-40</y>
      <width>851</width>
      <height>561</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 100%;
    height: 100%;
    background-color: rgba(0, 0, 0, 0.5); /* Nền đen với độ trong suốt 50% */
    display: flex;
    justify-content: center;
    align-items: center;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/hinh-nen-slide-dep-chu-de-am-nhac-de-thuong.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>361</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">height: 100%;
    background-color: rgba(0, 0, 0,); /* Nền đen với độ trong suốt 50% */
    display: flex;
    justify-content: center;
    align-items: center;
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>label_12</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
